code/modules/SCP/SCPs/SCP-457.dm

```diff
@@ -0,0 +1,233 @@
+GLOBAL_LIST_EMPTY(scp457s)
+
+/datum/scp/scp_457
+	name = "SCP-457"
+	designation = "457"
+	classification = KETER
+
+/mob/living/scp_457
+	name = "SCP-457"
+	desc = "A horrible band of fire, it somehow keeps a humanoid shape."
+	icon = 'icons/SCP/scp-457.dmi'
+	icon_state = "fireguy"
+	SCP = /datum/scp/scp_457
+	status_flags = NO_ANTAG
+	var/door_cooldown
+	health = 500
+	maxHealth = 500
+	var/next_emote = -1
+
+	see_invisible = SEE_INVISIBLE_NOLIGHTING
+	see_in_dark = 7
+
+	can_pull_size = 3
+	a_intent = "harm"
+	can_be_buckled = FALSE
+
+	var/aflame_cooldown
+	var/aflame_cooldown_time = 1.8 SECONDS
+
+	var/area/spawn_area
+
+/mob/living/scp_457/Initialize()
+	GLOB.scp457s += src
+	spawn_area = get_area(src)
+	add_language(LANGUAGE_EAL, FALSE)
+	add_language(LANGUAGE_SKRELLIAN, FALSE)
+	add_language(LANGUAGE_GUTTER, FALSE)
+	add_language(LANGUAGE_SIGN, FALSE)
+	add_language(LANGUAGE_ENGLISH, TRUE)
+	set_light(0.8, 0.3, 5, l_color = COLOR_ORANGE) //makes 457 emit light
+	add_verb(src, list(
+		/mob/living/scp_457/proc/burn,
+		/mob/living/scp_457/proc/hungryfuel,
+		/mob/living/scp_457/proc/taunt1,
+		/mob/living/scp_457/proc/taunt2,
+		/mob/living/scp_457/proc/taunt3,
+		/mob/living/scp_457/proc/no,
+		/mob/living/scp_457/proc/yes,
+		/mob/living/scp_457/proc/checkhealth,
+	))
+
+	. = ..()
+	return
+
+/mob/living/scp_457/Destroy()
+	GLOB.scp457s -= src
+	. = ..()
+	return
+
+/mob/living/scp_457/UnarmedAttack(atom/A)
+	var/mob/living/carbon/human/H = A
+	if(!istype(H))
+		return
+	if(H.SCP)
+		to_chat(src, "<span class='warning'><I>[H] is a fellow SCP!</I></span>")
+		return
+	if(H.stat == DEAD)
+		to_chat(src, "<span class='warning'><I>[H] is dead, it no longer can provide you with fuel.</I></span>")
+		return
+	if(istype(H))
```

Comment:
you literally checked for istype at line 62, you don't need to do it again

Reply:
i dont need to but i want to :trollface: